docs/creating-pdf-documents.md

```diff
@@ -107,6 +129,14 @@ exports:
     output: exports/my-document.pdf
 ---
 ```
+Finally, build the PDF output with Typst using the following command:
+
+```bash
+myst build --typst
```

Comment:
For now, I'm going to update this for the single-document workflow despite the points above -- we'll iterate at some point to cover `myst.yml`
```suggestion
myst build article.md --typst
```